logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/message/GetNotificationsUseCaseTest.kt

```diff
@@ -341,7 +356,42 @@ class GetNotificationsUseCaseTest {
         }
     }
 
+    @Test
+    fun givenConnectionRequests_thenNotificationListWithConnectionRequestMessage() = runTest {
```

Comment:
Maybe write another test to hide connection request from notifications when some one cancel it?

Reply:
there is no need for such test, as for now all the ConnectionRequests that user should be notified about are got by DB-query. 
So there is nothing to test in that UseCase in fact